packages/cohort_sdk/src/model/mod.rs

```diff
@@ -84,8 +84,8 @@ pub struct Config {
     pub agent: String,
     pub cohort: String,
     // The size of internal buffer for candidates
-    pub buffer_size: usize,
-    pub timeout_ms: u64,
+    pub buffer_size: u32,
```

Comment:
I will be addressing this in the separate PR. The size of u32 might not be enough.